src/Form/DataTransformer/ModelsToArrayTransformer.php

```diff
@@ -141,13 +151,18 @@ public function transform($collection)
         return $array;
     }
 
+    /**
+     * @return Collection<int|string, object>
+     *
+     * @phpstan-return Collection<array-key, T>
+     */
     public function reverseTransform($keys)
     {
         if (!\is_array($keys)) {
```

Comment:
Similar to https://github.com/sonata-project/SonataAdminBundle/pull/6377/files#r483949339,

The Symfony phpdoc say that we must be able to deal with empty values, should we add a check then ?

Reply:
Fixed it